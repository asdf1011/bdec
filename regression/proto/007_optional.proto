message optional {
    optional string a = 1;
    optional int32 b = 2;
    optional int32 c = 3;
}
