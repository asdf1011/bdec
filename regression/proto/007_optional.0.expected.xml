<optional>
    <b>5</b>
    <c>9</c>
</optional>
